app/resources/locale/en/en.po

```diff
@@ -559,139 +569,133 @@ msgid "Hexadecimal"
 msgstr "Hexadecimal"
 
 #: app/views/version.html:49
-#, fuzzy
 msgid "Icestudio is part of"
-msgstr "Icestudio Inception"
+msgstr "Icestudio is part of"
 
-#: app/scripts/services/graph.js:1619
+#: app/scripts/services/graph.js:1659
 msgid ""
 "If you have blank IN/OUT pins, it's because there is no equivalent in this "
 "board"
 msgstr ""
 "If you see blank IN/OUT pins, it is because there is no equivalent in this "
 "board"
```

Comment:
Could be
"If you see blank IN/OUT pins, it is because equivalent pins do not exist on this "
"board"

Reply:
👌